# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/ipc/RemoteDecoderModule.h
# Commit: 84b82a60dc74
# Full Hash: 84b82a60dc74098efc16c71e8efd85689d2ea680
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2020-10-21 09:51:28
# Regressor Bug: 1595994
# File Overlap Count: 1
# Description:
#   Bug 1595994 - P11. Don't unnecessarily create the RemoteDecoderManagerChild thread. r=mattwoodrow
#   
#   This is no longer required now that the RemoteDecoderManagerChild manages decoder creation
#   
#   Depends on D56857
# ==============================================================================

diff -r d82b6476a18d -r 84b82a60dc74 dom/media/ipc/RemoteDecoderModule.h
--- a/dom/media/ipc/RemoteDecoderModule.h	Tue Oct 20 23:27:21 2020 +0000
+++ b/dom/media/ipc/RemoteDecoderModule.h	Tue Oct 20 23:27:36 2020 +0000
@@ -17,7 +17,6 @@
   friend already_AddRefed<T> MakeAndAddRef(Args&&...);
 
  public:
-  static void Init();
   static already_AddRefed<PlatformDecoderModule> Create(
       RemoteDecodeIn aLocation);
 